tests/functional/n-window/02-big-window.t

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+# Test large window size using graphql and find tasks in window.
+# This is helpful with coverage by using most the no-rewalk mechanics.
```

Comment:
These show tests are integration testable which removes the potential for race conditions and gives us more flexibility to push tasks into whatever state we choose.

```python
# untested
from cylc.flow.scripts import show
from cylc.flow.option_parsers import Options
from cylc.flow.cyclers.integer import IntegerPoint
from cylc.flow.task_status import TASK_STATUS_SUCCEEDED

ShowOptions = Options(get_option_parser())

def show_task(schd, task):
        return await show(
            schd.tokens.id,
            schd.tokens.duplicate(cycle='1', task=task),
            ShowOptions(json=True)
        )

async def test_whatevs(...):
   ...
    async with start(schd):
        itask = schd.pool.get_task(IntegerPoint(1), 'task-name')
        await schd.update_data_structure()

        assert await show_task(schd, 'task-name') == {...}

        itask.reset_state(TASK_STATUS_SUCCEEDED)
        schd.data_store_mgr.delta_task_status(itask)
        assert await show_task(schd, 'task-name') == {...}

Reply:
Is this functional test still needed now that integration tests have been added? Or do they cover different things?

Reply:
At min tests API and cylc-show on large windows, also larger window size than integration tests (I think)